lib/foreman/renderer.rb

```diff
@@ -54,7 +54,7 @@ def foreman_url(action = "built")
 
       url_for :only_path => false, :controller => "/unattended", :action => action,
               :protocol  => protocol, :host => host, :port => port, :script_name => path,
-              :token     => (@host.token.value unless @host.token.nil?)
+              :token     => (@host.token.value unless @host.try(:token).nil?)
```

Comment:
don't understand this code. previous line has a local variable named host, this one has an instance variable... what's the difference?